style-guide.adoc

```diff
@@ -1516,60 +1527,61 @@ The table lists UI elements and the verbs and adjectives to use for them.
 
 |text area
 |enter
-|it's an input field, refer to it by its field name
+|It's an input field, refer to it by its field name
 
 |toggle 
 |use, switch on/off, activate/deactivate
-|some call it _switch_, users know hot it works, so you don't have to use verbs like _toggle_ or _slide_.
+|Some call it _switch_, users know hot it works, so you don't have to use verbs like _toggle_ or _slide_.
 
 |===
 
-IMPORTANT: For UI elements without interaction, use _is displayed_. It's OK to use the passive voice here because the agent is obvious. Avoid the following words: _displays_, _appears_, _opens_, or _shows_.
+IMPORTANT: For UI elements without interaction, use _is displayed_. It's OK to use the passive voice here because the agent is obvious. Avoid the following words for UI elements: _displays_, _appears_, _opens_, or _shows_.
 
-NOTE: For end user documentation, refer to unavailable commands and options on the UI as _unavailable_, not as _grayed out_, _dimmed_, _disabled_, or _inactive_. 
+NOTE: In end user documentation, refer to displayed but nonresponsive commands and options on the UI as _unavailable_, not as _grayed out_, _dimmed_, _disabled_, or _inactive_. 
 
 === Other user interface elements
 
-The following sections list software-specific UI elements of mobile apps. 
+The following sections list software-specific user interface elements of mobile apps. 
+
+==== Android-specific user interface elements
 
-=== Android user interface elements
+Check out the link:https://material.io/[Google Material Design resources] for the full list. 
 
-Check out Google material.io resources. 
+- *Bottom navigation bars:* Allow movement between primary destinations (3 to 5) in an app (mobile or tablet only). They consist of a container with icons that have text labels. Icons in the bottom navigation bars can be active or inactive. (note difference with unavailable). States: active, inactive, focused, or pressed.
 
-*Bottom navigation bars:* Allow movement between primary destinations (3 to 5) in an app (mobile or tablet only). They consist of a container with icons that have text labels. Icons in the bottom navigation bars can be active or inactive. (note difference with unavailable). States: active, inactive, focused, or pressed.
+- *Badges:* Bottom navigation icons can include badges in their upper right corner. These badges can contain dynamic information, such as a number of pending requests.
 
-*Badges:* Bottom navigation icons can include badges in their upper right corner. These badges can contain dynamic information, such as a number of pending requests.
+- *Top navigation bars:* Can contain menu icon (Hamburger), up arrow, back arrow, title, action items, and the overflow menu (3 dots)
```

Comment:
*Top navigation bars:* Can contain a menu icon (Hamburger), up arrow, back arrow, title, action items, and the overflow menu (3 dots).